src/main/java/com/adobe/aem/compgenerator/utils/ComponentUtils.java

```diff
@@ -130,7 +153,12 @@ private void createFolderWithContentXML(String path, String folderType)
                 rootElement.setAttribute("componentGroup", generationConfig.getGroup());
             } else if (folderType.equalsIgnoreCase(Constants.TYPE_SLING_FOLDER)) {
                 rootElement.setAttribute(Constants.JCR_PRIMARY_TYPE, folderType);
+            } else if (folderType.equalsIgnoreCase(Constants.TYPE_CLIENTLIB_FOLDER)) {
+                rootElement.setAttribute(Constants.JCR_PRIMARY_TYPE, folderType);
+                rootElement.setAttribute("categories", "[components." + generationConfig.getName() + "]");
```

Comment:
I wonder if we should use the `component-path` setting, replacing `/` with `.` rather than just `components` so that we get something like `newco.components.demo-comp` - my concerns with just `components.demo-comp` is it has no indication of what package/company it comes from and almost feels so generic that it could be misunderstood as OOTB.